lib/rust/ensogl/component/grid-view/src/column_widths.rs

```diff
@@ -0,0 +1,91 @@
+//! Storage for column's widths in the GridView. See [`ColumnWidths`].
+
+use crate::prelude::*;
+use std::fmt::Formatter;
```

Comment:
```suggestion
use crate::prelude::*;

use std::fmt::Formatter;
```
I think it could be fixed with `cargo run -p enso-formatter`.

Reply:
Fixed, thanks